azkaban_cli/azkaban_cli.py

```diff
@@ -224,6 +225,15 @@ def __fetch_projects(ctx, user):
     except FetchProjectsError as e:
         logging.error(str(e))
 
+@login_required
+def __fetch_sla(ctx, schedule):
+    azkaban = ctx.obj[u'azkaban']
+
+    try:
+        data = azkaban.fetch_sla(schedule)
```

Comment:
You're capturing the response, but not doing anything with it (like logging it to the console).

The response will be something like this when it succeeds:
```{'slaEmails': ['my.email@test.company.com'], 'allJobNames': ['job_1, 'job_2, 'job_3'], 'settings': [{'duration': '300m', 'rule': 'SUCCESS', 'id': 'test_id', 'actions': ['EMAIL']}]}```

I suggest you add a parsing function that will parse the response and print (`logging.info`) the output in a more human-readable fashion.

Take a look at [this PR here](https://github.com/globocom/azkaban-cli/pull/37/files#diff-4f86e3e923f34200240a9d0374b9c2c7), where I implement something very similar to what you want. Let me know if you have any questions